tests/tests/deposit_relay.rs

```diff
@@ -1,361 +0,0 @@
-extern crate bridge;
```

Comment:
why is it removed?

Reply:
those should indeed be commented in again and adapted to the changes. will do

Reply:
i now remember that i wanted to replace them with tests for the individual relays that are more focused

Reply:
done